wpilibj/src/main/java/edu/wpi/first/wpilibj/Compressor.java

```diff
@@ -76,28 +76,6 @@ public void close() {
     m_module = null;
   }
 
-  /**
-   * Start the compressor running in closed loop control mode.
-   *
-   * <p>Use the method in cases where you would like to manually stop and start the compressor for
-   * applications such as conserving battery or making sure that the compressor motor doesn't start
-   * during critical operations.
-   */
-  public void start() {
-    setClosedLoopControl(true);
-  }
-
-  /**
-   * Stop the compressor from running in closed loop control mode.
-   *
-   * <p>Use the method in cases where you would like to manually stop and start the compressor for
-   * applications such as conserving battery or making sure that the compressor motor doesn't start
-   * during critical operations.
-   */
-  public void stop() {
-    setClosedLoopControl(false);
-  }
-
```

Comment:
Can we add shims for these and deprecate, rather than removing?  It's quite likely there is team code that uses these.